### PR TITLE
feat: #tno-1558 - connect to PROD oracle instance

### PR DESCRIPTION
**Pull request by kylgarmor**
- added logging on main query
- fixed sort issue and use of lastModified instead of published

### CODE REVIEW

File: services/net/contentmigration/ContentMigrationAction.cs [hunk 1]
```diff
@@ -153,17 +161,35 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
...
-        DateTime? creationDateOfLastImport = !string.IsNullOrEmpty(manager.Ingest.GetConfigurationValue("creationDateOfLastImport")) ? manager.Ingest.GetConfigurationValue<DateTime>("creationDateOfLastImport") : null;
-        double migrationTimeOffsetInHours = !string.IsNullOrEmpty(manager.Ingest.GetConfigurationValue("migrationTimeOffsetInHours")) ? manager.Ingest.GetConfigurationValue<double>("migrationTimeOffsetInHours") : 0;
+        DateTime? creationDateOfLastImport = null;
+        try
+        {
+            creationDateOfLastImport = manager.Ingest.GetConfigurationValue<DateTime>("creationDateOfLastImport");
+        }
+        catch (Exception)
+        {
+            // migration has never been run before.
```
Comment (Fosol) [inline services/net/contentmigration/ContentMigrationAction.cs:171]: Ignoring exceptions is always risky behaviour.  You should almost always log.

File: services/net/contentmigration/ContentMigrationAction.cs [hunk 2]
```diff
@@ -153,17 +161,35 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
...
+        }
+        double migrationTimeOffsetInHours = 0;
+        try
+        {
+            migrationTimeOffsetInHours = manager.Ingest.GetConfigurationValue<double>("migrationTimeOffsetInHours");
+        }
+        catch (Exception)
+        {
+            // migration has never been run before.
```
Comment (Fosol) [inline services/net/contentmigration/ContentMigrationAction.cs:180]: Ignoring exceptions is always risky behaviour.  You should almost always log.

**Review by Fosol — APPROVED** [review]

File: services/net/contentmigration/ContentMigrationAction.cs (hunk 1 above)
Comment (kylgarmor) [reply services/net/contentmigration/ContentMigrationAction.cs:171]: sorted

File: services/net/contentmigration/ContentMigrationAction.cs (hunk 2 above)
Comment (kylgarmor) [reply services/net/contentmigration/ContentMigrationAction.cs]: sorted

**Pull request merged**